src/daft-sql/src/modules/aggs.rs

```diff
@@ -78,6 +78,22 @@ impl SQLFunction for AggExpr {
 
 fn handle_count(inputs: &[FunctionArg], planner: &SQLPlanner) -> SQLPlannerResult<ExprRef> {
     Ok(match inputs {
+        // count(null) always returns 0
+        [FunctionArg::Unnamed(FunctionArgExpr::Expr(SQLExpr::Value(Value::Null)))] => {
+            lit(0).alias("count")
```

Comment:
Could we add some tests for this along with group-bys? I want to ensure introducing a scalar expression here behaves ok.  I believe `select 0 from t group by x;` should be fine, just want to test. Thanks.